.env

```diff
@@ -11,3 +11,4 @@ MINER_PUBLIC_KEY=f89ef46927f506c70b6a58fd322450a936311dc6ac91f4ec3d8ef949608dbf1
 MINER_PRIVATE_KEY=da81490c7efd5a95398a3846fa57fd17339bdf1b941d102f2d3217ad29785ff0
 
 DISABLE_POW_VALIDATION=1
+DEVELOPMENT_BLOCK_RATE=1
```

Comment:
I think the default should match the default of the production networks, which is 30 seconds.